test/setup-beam.test.js

```diff
@@ -540,6 +540,13 @@ rebar ${rebar3}`
   simulateInput('elixir-version', elixirVersion)
   simulateInput('gleam-version', gleamVersion)
   simulateInput('rebar3-version', rebar3Version)
+
+  // test that elixir ref:v syntax works
+  toolVersions = `erlang ${erlang}\nelixr ref:v${elixir}`
+  fs.writeFileSync(filename, toolVersions)
+  appVersions = setupBeam.parseVersionFile(filename)
+  assert.strictEqual(appVersions.get('erlang'), erlang)
+  assert.strictEqual(appVersions.get('elixir'), elixir)
```

Comment:
It would be preferable that you changed the `toolVersions` constant, above, instead of adding to it and rewriting the file, unless there's a good reason for this I'm not seeing. Also, it seems there's a typo: `elixr` instead of `elixir`.

Reply:
You can replace `elixir ${elixir}` with `elixir ref:${elixir}` (no `v`), no problem. This way we test the optional `v` and your changes in a single go, what do you say?

Reply:
but then we don't test that it also works without the `ref:` right?

Reply:
That constant is

```
erlang   ${erlang}# comment, no space
elixir ${elixir}  # comment, with space
 not-gleam 0.23 # not picked up
gleam ${gleam} 
rebar ${rebar3}`
```

so there's tests for "with `ref:`" and "without `ref:`".